lib/art_vandelay.rb

```diff
@@ -116,11 +146,11 @@ def model_name
       records.model_name.name
     end
 
-    def row(attributes)
+    def row(attributes, format: :array)
       if self.attributes.any?
-        filtered_values(attributes.slice(*standardized_attributes))
+        filtered_values(attributes.slice(*standardized_attributes), format: format)
```

Comment:
[Minor]

We can actually omit the hash value thanks to [this feature](https://bugs.ruby-lang.org/issues/14579).

```suggestion
        filtered_values(attributes.slice(*standardized_attributes), format:)
```

Reply:
Nice! I'm not sure I've ever seen this in the wild. 👀